REVIEWING.rst

```diff
@@ -0,0 +1,37 @@
+Review Aiven Developer pull requests
+====================================
+
+We have many contributors and reviewers, so here's a checklist to share the things we want to look at when reviewing contributions. Please follow these guidelines when reviewing any pull request on Aiven Developer, and don't merge your own work.
+
+At an overall level
+-------------------
+
+#. Is this a migration from an old knowledgebase article? Make sure to add the old and new URLs to the migrations sheet when merging.
+
+#. If this addresses an open issue, have we added the magic ``Fixes #123`` incantation to the pull request description? (and if it doesn't address an issue, why not?)
```

Comment:
What should we do if there is no issue for a certain topic? Open an issue or?

Reply:
I don't think we need to do anything. But it would be good to have the issues as a way to feed the work that we're about to do.